.github/workflows/release.yml

```diff
@@ -74,8 +74,12 @@ jobs:
         include:
         - os: ubuntu-latest
           arch: linux-x86_64
+        - os: ubuntu-latest
+          arch: linux-arm64
         - os: macos-12
           arch: osx-x86_64
+        - os: macos-12
+          arch: osx-arm64
     runs-on: ${{matrix.os}}
```

Comment:
The `arch` variable doesn't seem to impact the actual architecture used to run here. It only impacts the filename of the generated artifact. Wouldn't this just build an x86_64 arch but name the file `arm64`? Since this change runs only on release, are you able to somehow verify that this is expected to work in some test environment before the PR gets merged?

Reply:
OK, I'll check and submit changes in two-three days.